yoga/Yoga.c

```diff
@@ -379,6 +379,14 @@ static void YGNodeMarkDirtyInternal(const YGNodeRef node) {
   }
 }
 
+static void YGNodeMarkDirtyRecursive(const YGNodeRef node) {
```

Comment:
I renamed it back, and also changed that it only affects the children and not the node itself, so we won't have the confusion with ```YGNodeMarkDirtyInternal```